script/test-unit

```diff
@@ -11,7 +11,7 @@ GREEN=$'\033[32m'
 TEXTRESET=$'\033[0m' # reset the foreground colour
 
 # -failfast -timeout=5m
-TESTFLAGS=(-cover "-coverprofile=cover.out" "${TESTFLAGS}")
+TESTFLAGS=(-test.short=true -cover "-coverprofile=cover.out" "${TESTFLAGS}")
```

Comment:
Is this change intended to be committed?

Reply:
We need to think about what behaviour we want where, but ultimately, I think yes, we want some switch around test.short=true , so one can easily skip the couple of long-ass tests I've added in this PR.
Question is, where/when do we want to disable them? on Semaphore? on localhost? always except when specified otherwise? etc.

Reply:
As discussed with others, we'll come back to that question later if we feel like we need to. I just wanted to raise attention to the fact that some of the added tests were long-lived, which is now done, so I'll remove that part for now.